src/include/configuration/configuration.h

```diff
@@ -2,17 +2,33 @@
 //
 //                         Peloton
 //
-// config.h
+// configuration.h
 //
-// Identification: src/include/common/config.h
+// Identification: src/include/configuration/configuration.h
 //
-// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+// Copyright (c) 2015-2017, Carnegie Mellon University Database Group
 //
 //===----------------------------------------------------------------------===//
-
 #pragma once
 
-#include <gflags/gflags.h>
+// internal use only in
+// configuration/configuration_manager.cpp
+#ifdef __PELOTON_CONFIG__
```

Comment:
I just want to make this header file seem to be independent and easy to use. I don't want to define one parameter twice. If I don't do so, I may need to define a parameter with gflag's DEFINE_* as well as ConfigurationManager::DefineConfig.